libsolidity/ast/Types.cpp

```diff
@@ -3754,6 +3786,34 @@ MemberList::MemberMap TypeType::nativeMembers(ASTNode const* _currentScope) cons
 		for (ASTPointer<EnumValue> const& enumValue: enumDef.members())
 			members.emplace_back(enumValue.get(), enumType);
 	}
+	else if (m_actualType->category() == Category::UserDefinedValueType)
+	{
+		auto& userDefined = dynamic_cast<UserDefinedValueType const&>(*m_actualType);
```

Comment:
```suggestion
		auto const& userDefined = dynamic_cast<UserDefinedValueType const&>(*m_actualType);
```

Reply:
But the `const` here is redundant, isn't it?

Reply:
It is, but may be nice to mention it - we usually do.